esp/templates/customforms/single_fieldset.html

```diff
@@ -18,15 +18,16 @@ <h2 class="sect_title">{{ fieldset.legend }}</h2>
                 <div class="dummy_field_text">{{ field.help_text }}</div>
             {% else %}
 				
-                {{ field.label_tag }}
+                <label for="id_{{ field.name }}" class="no_float">{{ field.label }}</label>
```

Comment:
Let's just turn this and everything below into template overrides, or delete if there's already a template override, and then deal with it from there. There's a change in the next release that will make those easier to manage.